src/serverdlg.cpp

```diff
@@ -193,7 +193,7 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
         SystemTrayIcon.setContextMenu ( pSystemTrayIconMenu );
 
         // set tool text
-        SystemTrayIcon.setToolTip ( QString ( APP_NAME ) + tr ( " server" ) );
+        SystemTrayIcon.setToolTip ( tr ( "%1 server", "%1 is the name of the main application" ).arg ( APP_NAME ) );
```

Comment:
Do we need the explanation here?

Maybe better make it clear that %1 is a placeholder for the application name (which will be filled automatically) —> no need for the translator to translate it.

Reply:
Translators will only see "%1 server" which can be confused if no context is provided, it can be helpful if we switch to other translation platforms like Transifex, Crowdin, or Weblate later. I guess it's okay for the current translation workflow (translator will need to clone the repo and Linguist can provide contexts like UI or code.